### ISSUE TITLE
Add LimitNOFILE increase to Role (Libreoffice not running Error)

### ISSUE THREAD

**Issue by Subdarub** [ebbba-org/ansible-role-bigbluebutton#89]
Ive based my ansible role on this ansible role.

Yesterday i realized that my role just wouldn't work properly with my bigger servers (32 cores / 64 Threads).
I was always getting the Libreoffice not running Error or Tomcat 7 + Libreoffice not running.

After some further investigation and comparison with my other big servers.

I found the Problem.

As a default bbb-web.service runs with a **LimitNOFILE** of **1024**, that's not enough for a server with that many cpu cores.
To circumvent this problem bbb-install.sh creates an **override.conf** for the service with a **LimitNOFILE** of **8192** , conditioned on the server having more than 8 CPUS.

Section of bbb-install.sh that does the above:

```
check_LimitNOFILE() {
  CPU=$(nproc --all)

  if [ "$CPU" -ge 8 ]; then
    if [ -f /lib/systemd/system/bbb-web.service ]; then
      # Let's create an override file to increase the number of LimitNOFILE 
      mkdir -p /etc/systemd/system/bbb-web.service.d/
      cat > /etc/systemd/system/bbb-web.service.d/override.conf << HERE
[Service]
LimitNOFILE=8192
HERE
      systemctl daemon-reload
    fi
  fi
}
```

Would be awesome if you could add this to the role, so the next guy with a bigger server, wont run into this issue.

**Comment by BrutalBirdie** [comment]
Is this still relevant since Libreoffice is now running as a docker container? @Subdarub

**Comment by Subdarub** [comment]
@BrutalBirdie This is change is not necessarily only for libreoffice. The override may be needed for other services included in the bbb-web.service stack. 

The Increase is still part of the official bbb-install.sh script as of today (Line: 685->), so i would assume it is still needed.

**Comment by crosscodr** [comment]
This was fixed in 5fb1c7c5
